drivers/counter/Kconfig.nrfx

```
@@ -12,7 +12,8 @@ config COUNTER_NRF_RTC
 config COUNTER_NRFX_TIMER_USE_CLOCK_CONTROL
 	def_bool y
 	depends on $(dt_nodelabel_enabled,timer120) || \
-		 $(dt_nodelabel_enabled,timer121)
+		   $(dt_nodelabel_enabled,timer121)
+	depends on !SOC_NRF54H20_CPUFLPR && !SOC_NRF54H20_CPUPPR
```

timer120 or timer121 cannot be used on PPR so that entry is redundant.
```suggestion
	depends on !SOC_NRF54H20_CPUFLPR
```